src/WebJobs.Script/Utility.cs

```diff
@@ -893,6 +893,11 @@ public static string GetSasTokenExpirationDate(string valueToParse, bool isAzure
             }
             else
             {
+                var isUri = Uri.IsWellFormedUriString(valueToParse, UriKind.Absolute);
+                if (!isUri)
```

Comment:
nit: isUri isn;t used anywhere else so we can just use it in the if- directly

```suggestion
                if (!Uri.IsWellFormedUriString(valueToParse, UriKind.Absolute);)
```